docs/beast_development.rst

```diff
@@ -420,4 +105,4 @@ Command to create it:
 
     .. code-block:: console
 
-        $ gource -s .06 -1280x720 --auto-skip-seconds .1 --multi-sampling  --stop-at-end --key --highlight-users --hide mouse,progress --file-idle-time 0 --max-files 0  --background-colour 000000 --font-size 22 --title "This is beast" --output-ppm-stream - --output-framerate 30 | avconv -y -r 30 -f image2pipe -vcodec ppm -i - -b 65536K beast_repo.mp4
+        $ gource -s .06 -1280x720 --auto-skip-seconds .1 --multi-sampling  --stop-at-end --key --highlight-users --hide mouse,progress --file-idle-time 0 --max-files 0  --background-colour 000000 --font-size 22 --title "This is ``beast``" --output-ppm-stream - --output-framerate 30 | avconv -y -r 30 -f image2pipe -vcodec ppm -i - -b 65536K ``beast``_repo.mp4

```

Comment:
Probably shouldn't add the quotes for the word beast in this command.  Not sure it will run.

Reply:
Quotes need removing.  Do not think the command will run with them.

Reply:
GaGHaaAgH, I've removed those quotes twice already, why do they keep reappearing... Okay. I've tried again.

Reply:
Hahahahaha....  Been there, done that!

Reply:
Still there.